### DISCUSSION TITLE
Hide menu on macOS

### DISCUSSION THREAD

**Discussion by VPKSoft** [tauri-apps/tauri#7150] [Q&A] [answered]
I have a custom menu implemented in the frontend side with custom title bar. How can I hide the application menu in macOS? In Linux and Windows the default menu is hidden.

![image](https://github.com/tauri-apps/tauri/assets/40712699/494bd249-ab17-4847-ac09-d7a2b68284e9)

**Comment by FabianLars** [comment, marked as answer]
you can use https://docs.rs/tauri/latest/tauri/struct.Builder.html#method.enable_macos_default_menu to disable the default menu, but keep in mind that the menu is required on macos for basic operations like copy&paste :/

**Reply by VPKSoft** [reply]
Works like a charm - thanks! I'm going to let the menu be however as the keyboard paste functionality flew away with removing the feature 🙂
